.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "linksf-dev": "vivid-inferno-4672"
```

Comment:
eh not really, but I don't think it hurts to have it. I will commit it out and put in a `.example` file I guess.